third_party/ruby/ruby.BUILD

```diff
@@ -1,23 +0,0 @@
-# vim: ft=bzl sw=4 ts=4 et
```

Comment:
And this file becomes unnecessary because of the `ruby_unpatched.BUILD` file?

Reply:
Yes, exactly. I favored our internal version of the ruby build scripts, as they were more up to date.